aspnetcore/security/anti-request-forgery.md

```diff
@@ -427,10 +427,10 @@ xhttp.onreadystatechange = function() {
         }
     }
 };
-xhttp.open('POST', '/api/password/changepassword', true);
+xhttp.open('POST', '/api/token/changeCode', true);
 xhttp.setRequestHeader("Content-type", "application/json");
 xhttp.setRequestHeader("X-CSRF-TOKEN", csrfToken);
-xhttp.send(JSON.stringify({ "newPassword": "ReallySecurePassword999$$$" }));
+xhttp.send(JSON.stringify({ "newCode": $CREDENTIAL_PLACEHOLDER$ }));
```

Comment:
OK for short term band aid, but  #21872 is the right way to fix this.

Reply:
@Rick-Anderson, it is not clear to me if you want this to merge now and follow  #21872 for an update later, or if this PR should wait.